src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants.java

```diff
@@ -2633,11 +2633,11 @@ public interface PrefsConstants extends com.google.gwt.i18n.client.Messages {
     String pythonPreferencesText();
 
     /**
-     * Translated "(NOTE: This project has already been configured with its own Python interpreter. Use the Project Options dialog to change the version of Python used in this project.".
+     * Translated "(NOTE: This project has already been configured with its own Python interpreter. Use the Edit Project Preferences button to change the version of Python used in this project.".
```

Comment:
The Project Options dialog (and the menu item that launches it) is named "Project Options", not "Project Preferences". Would it be more consistent if the button said "Edit Project Options..." instead of "Preferences"?

We call them preferences internally, but the UI generally calls them options.

Not specific to this change, just an observation.

Reply:
Thanks for calling that out! I was getting confused by that, I can update in both places for consistency

Reply:
Plus "options" is shorter so the button won't be quite as big.